nilearn/plotting/tests/test_img_plotting/test_img_plotting.py

```diff
@@ -93,20 +85,6 @@ def test_plot_functions_invalid_threshold(plot_func, img_3d_mni, tmp_path):
     plt.close()
 
 
-@pytest.mark.parametrize("plot_func", PLOTTING_FUNCS_3D)
-@pytest.mark.parametrize("cbar_tick_format", ["%f", "%i"])
-def test_cbar_tick_format(plot_func, img_3d_mni, cbar_tick_format, tmp_path):
-    """Test different colorbar tick format with 3D plotting functions."""
-    filename = tmp_path / "temp.png"
-    plot_func(
-        img_3d_mni,
-        output_file=filename,
-        colorbar=True,
-        cbar_tick_format=cbar_tick_format,
-    )
-    plt.close()
-
-
 def test_plot_carpet_default_params(img_4d_mni, img_3d_ones_mni, tmp_path):
```

Comment:
I am just wondering why we still need some of the "smoke tests" without the image compare decorator here. Maybe you have yet to remove them?

For example, this one seems like a copy of the one in L136 of `nilearn/plotting/tests/test_baseline_comparisons.py`.

Reply:
I have not been completely thorough in cleaning / moving tests so expect some duplicate.

Just want to make sure we want this before 'making ~perfect~ better'

Reply:
Ok makes sense!

Reply:
Also if we are ok using this for testing, we will have follow up PR to apply it to matrix plotting and other plotting functions...

Reply:
My first thought was that probably just examining/comparing Figure objects could be possible (without saving pngs).

I looked into how matplotlib does/suggests to do the testing, and they seem to have two methods:
- One that compares the "baselines" with PNGs: https://matplotlib.org/devdocs/devel/testing.html#writing-an-image-comparison-test
- But I think the second one seems to just compare Figure objects: https://matplotlib.org/devdocs/devel/testing.html#compare-two-methods-of-creating-an-image

Reply:
But that does not allow to have regression tests to check if a PR is modifying the behavior of a plotting function, no?

Reply:
Yeah looks like you can just give two Figure instances to compare if they are the same or not.